src/EnergyPlus/SingleDuct.cc

```diff
@@ -3130,12 +3130,12 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
                                          "User-Specified Constant Minimum Air Flow Fraction",
                                          MinAirFlowFracUser * this->ZoneTurndownMinAirFrac);
             if (state.dataGlobal->DisplayExtraWarnings) {
-                if ((std::abs(MinAirFlowFracDes - MinAirFlowFracUser) / MinAirFlowFracUser) > state.dataSize->AutoVsHardSizingThreshold) {
+                if ((MinAirFlowFracUser > 0.0) &&
+                    ((std::abs(MinAirFlowFracDes - MinAirFlowFracUser) / MinAirFlowFracUser) > state.dataSize->AutoVsHardSizingThreshold)) {
                     ShowMessage(state,
                                 format("SizeHVACSingleDuct: Potential issue with equipment sizing for {} = \"{}\".", this->sysType, this->SysName));
-                    ShowContinueError(state, format("User-Specified Minimum Cooling Air Flow Fraction of {:.5R} [m3/s]", MinAirFlowFracUser));
-                    ShowContinueError(state,
-                                      format("differs from Design Size Minimum Cooling Air Flow Fraction of {:.5R} [m3/s]", MinAirFlowFracDes));
+                    ShowContinueError(state, format("User-Specified Minimum Cooling Air Flow Fraction of {:.5R}", MinAirFlowFracUser));
+                    ShowContinueError(state, format("differs from Design Size Minimum Cooling Air Flow Fraction of {:.5R}", MinAirFlowFracDes));
```

Comment:
When MinAirFlowFracUser is 0.0, you have a divide by zero. Also, the error message has a copy-paste error since it includes units of m/s for this fraction (so dimensionless)

Reply:
OK, so if the air flow fraction is zero, this message just won't be emitted, right?  Is that what one would expect @rraustad ?  Perhaps this is something that is difficult or impossible to achieve using IDD min/max specs, but easy to sneak in here in the context of a unit test?